src/client/app/effects/snippet.ts

```diff
@@ -115,7 +116,10 @@ export class SnippetEffects {
     @Effect()
     loadSnippets$: Observable<Action> = this.actions$
         .ofType(Snippet.SnippetActionTypes.STORE_UPDATED, Snippet.SnippetActionTypes.LOAD_SNIPPETS)
-        .map(() => new Snippet.LoadSnippetsSuccessAction(storage.snippets.values()))
+        .map(() => {
+            trustedSnippetManager.reloadTrustedSnippets();
```

Comment:
is `reload` the right verb here?  Isn't it more like `cleanupTrustedSnippetsList`?

Reply:
Honestly doesn't make a difference as you are rebuilding that list anyways.

Reply:
Changed name.